### ISSUE TITLE
Parent Theme?

### ISSUE THREAD

**Issue by WPDevHQ** [SiteEditor/tanin#1]
Hello there,

Is the parent theme for this child theme available or is it a child of TwentySeventeen?

**Comment by parsaatef** [comment]
Hi dear,
Yes, there is a parent theme for Tanin, that is Twenty Seventeen Plus Lite but we don't release it for open Source.
Do you need to this theme?

**Comment by WPDevHQ** [comment]
Yes please. I'm starting to develope for your SiteEditor plugin so need as much insight as possible.

**Comment by parsaatef** [comment]
Please give me your email for send Theme. but please go to
plugins\site-editor\framework\includes\theme-support

for more information about developing your theme OR
We have a sample module for developers guide which you can find at “Site Editor Plugin Folder/editor/extensions/pagebuilder/modules/api-test-module”
you can test this module in our editor. It’s active by default.

We also have a sample options file for developers guide( you can use for theme options and plugin options in front end editor mode ) which you find at “Site Editor Plugin Folder/editor/extensions/options-engine/includes/demo/site-editor-sample-options.php”

Notice 1: To test this options in our editor you need to activate it on WordPress Admin > SiteEditor > General Settings > Developer Sample Options.

**Comment by WPDevHQ** [comment]
Awesome!

Please mail me at zulfnore@hotmail.co.uk, thanks.
